fMRIVolume/scripts/DistortionCorrectionAndEPIToT1wReg_FLIRTBBRAndFreeSurferBBRbased.sh

```diff
@@ -432,10 +437,56 @@ case $DistortionCorrection in
                 --smoothingfwhm="2" \
                 --inputdir="$WD"
         fi
-            
+
 
         ;;
 
+    ${NONE_METHOD_OPT})
+
+            log_Msg "---> No distortion correction"
+
+            log_Msg "---> FAKE WarpField"
+            ### faked warp image with three frames
+            ${FSLDIR}/bin/fslmaths ${T1wImage} -mul 0 ${WD}/WarpField.nii.gz
+            ${FSLDIR}/bin/fslmerge -t ${WD}/WarpField.nii.gz ${WD}/WarpField.nii.gz ${WD}/WarpField.nii.gz ${WD}/WarpField.nii.gz
+
+            log_Msg "---> FAKE Jacobian Volume"
+            ###Jacobian Volume FAKED for Regular Fieldmaps (all ones) ###
+            ${FSLDIR}/bin/fslmaths ${T1wImage} -mul 0 -add 1 -bin ${WD}/Jacobian.nii.gz
```

Comment:
Don't see why a fake jacobian volume is necessary either.

Reply:
Same reason. To make sure it is there if it is aa dependency later on in the code.

Reply:
The Jacobian is a bit more complicated, as I see that a jacobian of some sort is an "output" of the DistortionCorrection script, and thus might indeed be expected to exist as part of some downstream scripts.